data/brands/tourism/hotel.json

```diff
@@ -3637,6 +3637,19 @@
         "name:ja": "スーパーホテル",
         "tourism": "hotel"
       }
+    },
+      {
+     "displayName": "SureStay by Best Western",
+     "id": "",
+     "locationSet": {"include": ["au","ca","de","fr","gb","in","it","la","no","se","th","us"]},
+     "matchNames": ["SureStay, Sure Stay"],
+     "tags": {
+       "brand": "Best Western Hotels & Resorts",
+       "brand:wikidata": "Q830334",
```

Comment:
SureStay calls itself “SureStay by Best Western”, but it isn’t the same chain. (These days, it’s actually owned by Best Western’s corporate parent, BWH.)